### PR TITLE
fix for np.float32 serialization

### PR DESCRIPTION
**Pull request by santiatpml**
Added tests for transforms and moved tune to a different set.

### CODE REVIEW

File: pgml-extension/tests/test.sql [hunk 1]
```diff
@@ -27,3 +27,4 @@ SELECT pgml.load_dataset('wine');
 \i examples/multi_classification.sql
 \i examples/regression.sql
 \i examples/vectors.sql
+\i examples/transformers.sql
```
Comment (montanalow) [inline pgml-extension/tests/test.sql:30]: We don't run transformers or finetuning during tests, since they take a lot of RAM and download time.

**Review by montanalow — CHANGES_REQUESTED** [review]

File: pgml-extension/tests/test.sql (hunk 1 above)
Comment (santiatpml) [reply pgml-extension/tests/test.sql]: Removed them.

**Review by montanalow — APPROVED** [review]

File: pgml-extension/Dockerfile.local [hunk 2]
```diff
@@ -11,7 +11,7 @@ RUN cat /etc/apt/sources.list
 RUN apt-get update && apt-get install -y postgresql-pgml-14
 
 # Cache this, quicker
-RUN pip3 install xgboost scikit-learn diptest torch lightgbm transformers datasets sentencepiece sentence_transformers sacremoses sacrebleu rouge
+RUN pip3 install xgboost scikit-learn diptest torch lightgbm transformers datasets sentencepiece sentence_transformers sacremoses sacrebleu rouge protobuf
```
Comment (montanalow) [inline pgml-extension/Dockerfile.local:14]: we should not need protobuf, what requires that?

Comment (santiatpml) [reply pgml-extension/Dockerfile.local:14]: Finbert model needs protobuf.
```sql
SELECT pgml.transform(
    inputs => ARRAY[
        'Stocks rallied and the British pound gained.', 
        'Stocks making the biggest moves midday: Nvidia, Palantir and more'
    ],
    task => '{"task": "text-classification", 
              "model": "ProsusAI/finbert"
             }'::JSONB
) AS market_sentiment;
```

Comment (montanalow) [reply pgml-extension/Dockerfile.local:14]: wow... that's... indicative of a whole new level... are they actually calling any rpcs, or are they just serializing in a round about way? I don't really care...

Comment (santiatpml) [reply pgml-extension/Dockerfile.local]: I could not reproduce the error I saw before without protobuf on two other machines. Looks like we don't need it.

**Pull request merged**